### ISSUE TITLE
Things To Do

### ISSUE THREAD

**Issue by gcarl96** [AdamStein97/Football-Prediction#1]
Required:

* [ ] clean data
* [x] define dict for formation names
* [x] set up everything for training
* [x] fix cropping
* [ ] standardize input with scaler
* [x] image resizing
* [x] data for all seasons
* [x] config for model params
* [ ] produce images to visualise encoded match data
* [x] Save match matrices
* [ ] Sort out test set bug

Good to have:
* [x] config files
* [ ] copy across RPS and Brier Scores
* [ ] batch normalisation
* [ ] Add dropout
* [ ] Add l2
* [x] Load fifa 20 data
* [x] Fix missing players in Fifa data

Useful
* [ ] include goalkeepers
* [ ] extra features possible such as form
* [x] validation data
* [x] tensor board
* [ ] add betting odds field
